handbook/product/design/design_process.md

```diff
@@ -16,120 +16,139 @@ While asynchronous communication is a core attribute of remote work, key moments
-      - Prepare redlines or an interaction delivery writeup for the engineers. Consider the following:
-          - Margin and padding
-          - Animation
-          - Links and hover states
-      - Describe expected behavior of layout for tablet and moble screensizes
-        - Due to the low number of mobile and table visitors (< 3%) design comps for these sizes are generally not required 
-      - Schedule a meeting with engineers to discuss the interaction
-  - Tools
-      - Figma
-- Metrics collection:
-  - In the RFC process, the definition of success will include items which can be measured and evaluated
-  - Involve the business operations team in determining what metrics to track and how they are tracked
-  - Work with engineers to add the required eventLog tracking
-  - Tools
-      - Looker
-      - Google analytics
-- Refinement
-  - Designers will review metrics and determine if their designs are meeting objectives set for them.
-  - If areas for improvement are found, tickets will be created to document the change required to improve the issue
+- Process
+    - A GitHub issue should be created to track the work
+    - Designs are produced in Figma
+    - Designs will be announced in Slack and linked in the GitHub issue well before they are complete for review
+    - Ensure designs meet the visual design checklist:
+      - Ensure spacing is consistent and matches the 8pt grid system
+      - Ensure text, colors and other sytles match existing styles, if possible. 
+        - Introduce new text styles only if really necessary.
+        - If adding new styles, ensure that they meet the accessibility standards and add them to the design system 
+      - Review designs for accessability:
+        - Color blind check
+        - Contrast check should meet AA standard for small text
+    - Schedule and conduct a design presentation meeting if the size of the project requires
+    - Receive signoff from stakeholders
+    - If components were created in the visual design process, when the designs are signed off on, those components will be moved to the Sourcegraph component system
+    - Prepare redlines or an interaction delivery writeup for the engineers. Consider the following:
+        - Margin and padding
+        - Animation
+        - Links and hover states
+    - Describe expected behavior of layout for tablet and moble screensizes
+      - Due to the low number of mobile and table visitors (< 3%) design comps for these sizes are generally not required 
+    - Schedule a meeting with engineers to discuss the interaction
+- Tools
+    - Figma
+
+### Metrics collection
+
+- In the RFC process, the definition of success will include items which can be measured and evaluated
+- Involve the business operations team in determining what metrics to track and how they are tracked
+- Work with engineers to add the required eventLog tracking
+- Tools
+    - Looker
+    - Google analytics
```

Comment:
Metrics impact many stages of product development. This should probably live under `product/analytics` once fleshed out a bit more.